README.md

```diff
@@ -83,6 +100,31 @@ In this case caller should use
 associate this session with one of the previous handshakes to identify the connecting
 client.
 
+[Back to TOC](#table-of-contents)
+
+resty.kong.tls.set\_upstream\_cert\_and\_key
+---------------------------------------
```

Comment:
style: I notice we fixed the style of the above `resty.kong.tls.get_full_client_certificate_chain` header, however this header isn't properly styled - not a blocker for me, just pointing it out since I see the above fix.